src/carousel.tsx

```diff
@@ -287,14 +289,30 @@ export const CarouselPageControls = ({
     setShouldAutoplay,
     prevArrowEnabled,
     nextArrowEnabled,
+    pagesCount,
+    currentPageIndex,
 }: CarouselPageControlsProps): JSX.Element => {
     const {texts, t} = useTheme();
     const variant = useThemeVariant();
+    const prevPageNumberText =
```

Comment:
Perhaps we could add support to the translate function to accept a string: `token: Token | string`.

Inside the translate function:
```js
const text = typeof token === 'string' ? token : token[language] || token.en
```

This way we could write this code as:
```js
t(text.patata || tokens.patata, param1, param2)
```